js/modules/map.js

```diff
@@ -0,0 +1,68 @@
+import { disableForm, enableForm } from './disable-enable-forms.js';
+import { createAdvertisements } from './create-advertisment.js';
+import { createAdvertisementsMarkup } from './create-advertisements-markup.js';
+
+const address = document.querySelector('#address');
+const DEFAULT_COORDS = { lat: 35.652832, lng: 139.839478 };
+
+address.value = `${DEFAULT_COORDS.lat.toFixed(5)}, ${DEFAULT_COORDS.lng.toFixed(5)}`;
+
+disableForm();
+const map = L.map('map-canvas')
+  .on('load', () => {
+    enableForm();
+  })
+  .setView(DEFAULT_COORDS, 13);
+
+L.tileLayer(
+  'https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png',
+  {
+    attribution: '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors',
+  },
+).addTo(map);
+
+const mainPinIcon = L.icon({
+  iconUrl: './../img/main-pin.svg',
+  iconSize: [52, 52],
+  iconAnchor: [26, 52],
+});
+
+const advertisementMarkerIcon = L.icon({
+  iconUrl: './../img/pin.svg',
```

Comment:
```suggestion
  iconUrl: 'img/pin.svg',
```
А с такими путями картинки нормально отображаются?

Reply:
Да, проблем никаких вроде.